src/components/Books.js

```diff
@@ -1,36 +1,17 @@
-import React, { useState } from 'react';
-import PropTypes from 'prop-types';
+import React from 'react';
+import { useSelector } from 'react-redux';
+// eslint-disable-next-line import/no-cycle
```

Comment:
- [ Required ] I think it would be better if you fix this `Eslint` error, disabling linter errors is not considered a good practice in software development especially if it's done in code. If you have to disable the `eslint` rule it would be better if you do it from the `.eslintrc.json` file as shown in the example below.

## Example code
```JavaScript
{
  "env": {
    "browser": true,
    "es6": true,
    "jest": true
  },
  "parser": "@babel/eslint-parser",
  "parserOptions": {
    "ecmaFeatures": {
      "jsx": true
    },
    "ecmaVersion": 2018,
    "sourceType": "module"
  },
  "extends": ["airbnb", "plugin:react/recommended", "plugin:react-hooks/recommended"],
  "plugins": ["react"],
  "rules": {
    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
    "react/react-in-jsx-scope": "off",
    "import/no-unresolved": "off",
    "import/no-extraneous-dependencies": "off",
    "no-shadow": "off",
    "no-console": "off",
    "import/no-cycle": "off"
  },
  "overrides": [
    {
      "files": ["src/**/*Slice.js"],
      "rules": { "no-param-reassign": ["error", { "props": false }] }
    }
  ],
  "ignorePatterns": [
    "dist/",
    "build/"
  ]
}
```